exporter/loggingexporter/config.go

```diff
@@ -31,3 +31,10 @@ type Config struct {
 	// SamplingThereafter defines the sampling rate after the initial samples are logged.
 	SamplingThereafter int `mapstructure:"sampling_thereafter"`
 }
+
+var _ config.Exporter = (*Config)(nil)
+
+// Validate checks if the exporter configuration is valid
+func (cfg *Config) Validate() error {
+	return nil
+}
```

Comment:
Maybe checking if the logging level is valid here? We do something similar on the factory function:

https://github.com/open-telemetry/opentelemetry-collector/blob/4a6a25e033f6fce1bca027af5fc354180842cbc7/exporter/loggingexporter/factory.go#L89-L92